crates/ruff_python_formatter/tests/snapshots/black_compatibility@miscellaneous__force_pyi.py.snap

```diff
@@ -94,14 +93,14 @@ def eggs() -> Union[str, int]: ...
 +
 +class F(A, C):
 +    ...
++
++
++def spam() -> None:
++    ...
```

Comment:
Unrelated to this PR: interesting, black sometimes keeps the single statement formatting